Lesson16/Merve/script.js

```diff
@@ -0,0 +1,129 @@
+    } else if (isStudent === "yes") {
+        console.log("You are old enough to vote.");
+    }
+}
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+function jobOffer(salary, experience) {
+    if (salary < 30000 && experience < 5) {
+        console.log("Sorry, we cannot offer you the job at this time.");
+    } else if (salary < 50000 && experience >= 5 && experience <= 10) {
+        console.log("We can offer you the job at a lower salary.");
+    } else {
+        console.log("We can offer you the job at the requested salary.");
+    }
+}
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive), print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!".function checkTestScore(score) {
+    function checkTestScore(score) {
+        if (score < 60) {
+            console.log("You failed the test.");
+        } else if (score >= 60 && score <= 70) {
+            console.log("You passed the test, but your grade is not great.");
+        } else if (score >= 71 && score <= 80) {
+            console.log("You got a B.");
+        } else if (score >= 81 && score <= 90) {
+            console.log("You got an A.");
+        } else {
+            console.log("You got an A+ - Great job!");
+        }
+    }
+// 14. Write a function that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). Use a switch statement to handle the different cases.
+function checkGradeLetter(grade) {
+    switch (grade) {
+        case "A":
+        case "B":
+        case "C":
+            console.log("Passing grade");
+            break;
+        case "D":
+        case "F":
+            console.log("Failing grade");
+            break;
+        default:
+            console.log("Invalid grade");
+    }
+}
+// 15. Write a function that asks the user to enter their age, and then calculates and displays their age in dog years (where one human year is equivalent to seven dog years). If the user enters a negative number or zero, display an error message.
+function calculateDogYear(yourAge){
+    if(yourAge<0 || yourAge<=0){
```

Comment:
`yourAge<=0` covers `yourAge<0` so you don't need to have both checks in this case. Should be: `if(yourAge<=0){`